source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -354,3 +352,187 @@ to restart the MinIO service:
 
 Once the deployment is online, use :mc:`mc admin info` to confirm the
 uptime of all remaining servers in the deployment.
+
+.. _minio-decommission-multiple-pools:
+
+Decommission Multiple Server Pools
+----------------------------------
+
+.. versionchanged:: RELEASE.2023-01-18T04-36-38Z
+
+You can decommission multiple server pools from the same command.
+
+MinIO immediately stops write access to all decommissioning pools.
+Decommissioning happens one pool at a time.
+Each pool completes the decommission process before MinIO begins draining the next pool.
+
+To decommission multiple server pools from one command, add each server pool in a comma-separated list.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin decom start myminio/ /disk/xl{1...4},/disk/xl{5...8}
```

Comment:
I'd suggest sticking to the hostname syntax here - that would be more normal. e.g.

```suggestion
   mc admin decom start myminio/ \
      https://minio-{1...4}/drives/minio-{1...4}, \
      https://minio-{5...8}/drives/minio-{1...4}
```

maybe prefix this with

"The following command decommissions two pools in the specified deployment. The command specifies the full hostname and disk definition of that pool as used in the MinIO server startup settings. 

This command assumes the deployment has at least one remaining pool to route all decommissioned objects."

Reply:
Or drop this entirely, since the procedure covers it sufficiently. No need to shortcut.

Reply:
Dropped.